llvm/test/tools/repo-create-ticket/ticket-path.ll

```diff
@@ -0,0 +1,17 @@
+; REQUIRES: asserts
+; RUN: rm -rf %t && mkdir -p %t
+; RUN: env REPOFILE=%t/db.db llc -filetype=obj %s -o %t/t1.o
+; RUN: repo-ticket-dump %t/t1.o > %t/out1.txt
+; RUN: echo 'CHECK: ticket directory:%t' > %t/expected.txt
+; RUN: cat %t/out1.txt | repo-create-ticket -debug -debug-only repo-create-ticket -repo=%t/db.db -o %t/t2.o - 2>&1 | FileCheck %t/expected.txt
+
```

Comment:
To ignore the lit issue (changing upper case drive `C:` to lower case `c:`) ,  we could add `--ignore-case` to the command `FileCheck %t/expected.txt`.

Reply:
That would work, but some file systems are case-sensitive so isn’t a perfect solution. Do you know why the case of the drive letter is different?

Reply:
It mentioned in [D77496]( https://reviews.llvm.org/D77496), Lit converts config paths to lower case in discovery.py. This causes __file__ to be all lower-case (including the drive letter). I found in the function `getTestSuite` defined in the discovery.py file, it uses the `cfgpath = os.path.normcase(cfgpath)` (line 56).

Reply:
Thanks for tracking this down. With this in mind, I don’t think that I have much choice but to use the --ignore-case switch (commit f0aa4fa25e008f93f7566c68521be6ae7c31638c).